docs/en/internationalization.md

```diff
@@ -0,0 +1,132 @@
+# Contributing to internationalization (i18n)
+
+Thanks to different contributors, FreshRSS is actually translated in 15 languages. This section will explain you the basis of the internationalization in FreshRSS, either if you want to translate the application in your own language or if you have to make a specific change.
```

Comment:
Do we really want to mention a specific number? ;-) Also "actually" doesn't seem appropriate, it should be left out or replaced by something like "actively."

```suggestion
Thanks to our contributors, FreshRSS is translated into more than 15 languages. This section will explain the basics of internationalization in FreshRSS, from translating the application to your own language to making a specific change.
```

Reply:
> Do we really want to mention a specific number? ;-) 

I didn't want to but since it's translated exactly into 15 languages, I had some remorse to write "more than 15 languages"… But we just need to find someone to add another translation and we'll be good :p